charts/dnsmadeeasy-webhook/CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0.3]
+
+### Changed
+
+- Fix deprecation warnings
+
+## [1.0.1]
+
+### Changed
+
+- Fix for status probes
+
+## [1.0.0]
```

Comment:
Since this is a new chart, we can stick with 1.0.0 right?

Reply:
I just thought of this, but this file can be removed now, right? Since the new templates will put the changelog in the README file automagically...

Reply:
Oh, I see - yes, this is not needed anymore since we have the template - removing